numpy/core/src/multiarray/ctors.c

```diff
@@ -4152,7 +4155,22 @@ _array_fill_strides(npy_intp *strides, npy_intp const *dims, int nd, size_t item
                     int inflag, int *objflags)
 {
     int i;
+
 #if NPY_RELAXED_STRIDES_CHECKING
+    npy_bool empty = 0;
```

Comment:
Do we use `NP_TRUE` and `NPY_FALSE` with `npy_bool`?

Reply:
I wonder if we can just replace everything with `bool` now that we require C99... Obviously not in this patch.